controllers/script.js

```diff
@@ -246,11 +246,14 @@ var getScriptPageTasks = function (aOptions) {
 
   // Setup the flagging UI
   tasks.push(function (aCallback) {
-    // Can't flag when not logged in or when user owns the script.
-    if (!authedUser || aOptions.isOwner) {
-      aCallback();
-      return;
-    }
+    // WARNING: Possible duplicated code from flagLib.flaggable
+
+      // Can't flag when not logged in or when user owns the script.
+      if (!authedUser || aOptions.isOwner) {
```

Comment:
Additional recheck for direct route *(no UI)* found at [.../blob/`703df10`/libs/flag.js#L176](https://github.com/OpenUserJs/OpenUserJS.org/blob/703df104da6b3667687da8311c7c846d76bf9fc9/libs/flag.js#L176) and [.../blob/`703df10`/libs/remove.js#L313](https://github.com/OpenUserJs/OpenUserJS.org/blob/703df104da6b3667687da8311c7c846d76bf9fc9/libs/remove.js#L313) ... looks secure.